### ISSUE TITLE
Add missing DataFlowSanitizer instrumentation for function `fmodf`

### ISSUE THREAD

**Issue by bradlarsen** [trailofbits/polytracker#4]
Some PDFs cause `mutool draw` to emit the following output:
```
==6==WARNING: DataFlowSanitizer: call to uninstrumented function fmodf
```

**Comment by ThatsNotVeryCashMoneyOfYou** [comment]
This will be fixed in the C++ PR

**Comment by bradlarsen** [comment]
Seems to be fixed by the merge of PR #28.
